doc/flatpak-mask.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
+    "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+
+<refentry id="flatpak-mask">
+
+    <refentryinfo>
+        <title>flatpak mask</title>
+        <productname>flatpak</productname>
+
+        <authorgroup>
+            <author>
+                <contrib>Developer</contrib>
+                <firstname>Alexander</firstname>
+                <surname>Larsson</surname>
+                <email>alexl@redhat.com</email>
+            </author>
+        </authorgroup>
+    </refentryinfo>
+
+    <refmeta>
+        <refentrytitle>flatpak mask</refentrytitle>
+        <manvolnum>1</manvolnum>
+    </refmeta>
+
+    <refnamediv>
+        <refname>flatpak-mask</refname>
+        <refpurpose>Mask out updates and automatic installation</refpurpose>
+    </refnamediv>
+
+    <refsynopsisdiv>
+            <cmdsynopsis>
+                <command>flatpak mask</command>
+                <arg choice="opt" rep="repeat">OPTION</arg>
+                <arg choice="plain" rep="repeat">PATTERN</arg>
+            </cmdsynopsis>
+    </refsynopsisdiv>
+
+    <refsect1>
+        <title>Description</title>
+
+        <para>
+          Flatpak maintains a list of patterns that that define which refs are masked.
+          A masked ref will never be updated or automatically installed (for example an
+          masked extension marked auto-download will not be downloaded). You can still
+          manually install such refs, but once they are installed the version will be pinned.
```

Comment:
Just to make sure: you can manually install masked refs, but a manual update will not work ?

Reply:
Yeah, you can install it, but once its installed the current version is locked down. One could imagine being able to force-update with flatpak update if you specify the exact ref, but thats not going to work in general with libflatpak APIs, where you always specify a ref to update, so best be consistent here.